tests/integration/udf/test_count_dict_entropy.py

```diff
@@ -4,8 +4,10 @@
 import numpy as np
 import pytest
 
+from tests.source_types import SNOWFLAKE_SPARK_DATABRICKS
 
-@pytest.mark.parametrize("source_type", ["snowflake, spark"], indirect=True)
```

Comment:
having consistent variables will help to avoid typos like this - where the two source types should be different strings, and not in the same string.